app/models/post.rb

```diff
@@ -0,0 +1,3 @@
+class Post < ApplicationRecord
```

Comment:
Let's use validation for post title and body. So that user can't submit any empty post.